src/app/public/modules/colorpicker/colorpicker-input.directive.ts

```diff
@@ -134,7 +133,7 @@ export class SkyColorpickerInputDirective
 
   /* istanbul ignore next */
   @HostListener('blur')
-  public onBlur(event: any) {
+  public onBlur() {
```

Comment:
Angular's type checking fails if a `HostListener` has an event argument, but `['$event']` isn't specified. We're not using the event, so I removed it. 
See: https://stackoverflow.com/a/50409526/6178885